docs/reference/known_limitations.md

```diff
@@ -28,6 +38,31 @@ Let's delve into the limitations of on-demand backups in Percona Everest.
 
     If you have created two schedules using backup storage `bucket-1` and `bucket-2`, and an on-demand backup using backup storage `bucket-3`, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.
 
+### PSMDB backup failures
+
+There maybe instances when your MongoDB backups may encounter unexpected failures. 
+
+Let's check the reason for these failures by running the following command:
+
+    kubectl get psmdb-backup <BACKUP_NAME> -n <YOUR_NAMESPACE> -o yaml | grep error
+
+Here are some potential errors you could encounter:
+
+- `starting deadline exceeded`
+- `'couldn''t get response from all shards: convergeClusterWithTimeout:`
+
+#### Workarounds for PSMDB backup failures
```

Comment:
PSMDB isn't a well-known acronym for users, I think we should refer to MongoDB instead.
```suggestion
#### Workarounds for MongoDB backup failures
```